test/functional/mongo_client.test.js

```diff
@@ -831,4 +832,10 @@ describe('MongoClient', function() {
         }
       });
   });
+
+  it('should have readPreference accessor from options', function() {
```

Comment:
I think we can help our future selves a little here by improving this test name, maybe something like:
```suggestion
  it('should cache a resolved readPreference from options', function() {
```